src/zenml/model/__init__.py

```diff
@@ -12,23 +12,4 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 
-"""Initialization of ZenML model.
-ZenML model support Model Control Plane feature.
-"""
-
-from zenml.model.model_version import ModelVersion
-from zenml.model.artifact_config import (
-    DataArtifactConfig,
-    ModelArtifactConfig,
-    EndpointArtifactConfig,
-)
-from zenml.model.link_output_to_model import link_output_to_model
-
-
-__all__ = [
-    "DataArtifactConfig",
-    "EndpointArtifactConfig",
-    "ModelArtifactConfig",
-    "ModelVersion",
-    "link_output_to_model",
-]
+"""Concepts related to the Model Control Plane feature."""
```

Comment:
Two reasons:
- Main reason: it leads to all sorts of cyclic imports that I didn't know how to resolve otherwise
- We expose `zenml.model.ModelVersion` now as a top-level ZenML import `from zenml import ModelVersion` and if `zenml.models.__init__` was not empty, this would cause unnecessary chain-imports